### PR TITLE
added export GO111MODULE=on to go get example

### PR DESCRIPTION
**Pull request by mooperd**
https://github.com/kubermatic/kubeone/issues/742

Signed-off-by: Andrew Holway <andrew.holway@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ sudo mv kubeone /usr/local/bin
 The alternative way to install KubeOne is using `go get`.
 
 ```bash
-go get -u github.com/kubermatic/kubeone
+GO111MODULE=on go get -u github.com/kubermatic/kubeone
```
Comment (kron4eg) [inline README.md]:
```suggestion
GO111MODULE=on go get github.com/kubermatic/kubeone
```

**Comment by kron4eg** [comment]
@mooperd unfortunately git commit from the web is not compatible with our DCO policy. You'd need to `git commit --amend -s` your latest commit.

**Comment by mooperd** [comment]
Actually - I am quite lost how to make this compliant with your policy. My git-fu is not that powerful.

**Comment by kron4eg** [comment]
no worries, I've reposted it (in https://github.com/kubermatic/kubeone/pull/744) with some additions

**Pull request closed without merging**